src/modules/core/filter_brightness.c

```diff
@@ -72,6 +72,15 @@ static int sliced_proc(int id, int index, int jobs, void *cookie)
                     p += 4;
                 }
             }
+        } else if (ctx->image->format == mlt_image_rgba64) {
+            for (int line = 0; line < slice_height; line++) {
+                uint16_t *p = (uint16_t *) ctx->image->planes[0]
+                              + ((slice_line_start + line) * ctx->image->strides[0]) + 3;
+                for (int pixel = 0; pixel < ctx->image->width; pixel++) {
+                    *p = (*p * m) >> 16;
+                    p += 4;
```

Comment:
Use array indexing, but `p[pixel]` reads weird  using `pixel` for the index var name. I would change `line` to `row`, `p` to `line`, and `pixel` to `col` => `line[col]`